app/javascript/forms/form-buttons-reducer.js

```diff
@@ -0,0 +1,32 @@
+import FormButtons from './form-buttons';
+
+const REDUX_FORM_BUTTON_INIT = 'FormButtons.init';
```

Comment:
I don't like having those looooong constants everywhere.

Could we maybe have an object of actions instead?

Something like..

```
export const FormButtonActions = {
  init: 'FormButtons.init',
  customLabel: 'FormButtons.customLabel',
  ...
}
```

so that we can have use `something.actions.FormButtons.init` for example?